plugins/netmiko/plugin.spec.yaml

```diff
@@ -20,8 +23,24 @@ hub_tags:
   use_cases: [data_utility, threat_detection_and_response]
   keywords: [netmiko, SSH, network automation]
   features: []
+key_features:
+- Automate network configuration
+- Multiple SSH commands can be provided in one action
+sdk:
+  type: slim
+  version: 6.0.0
```

Comment:
Can we update this to 6.0.1?

Reply:
yeah literally just doing this now since the validator failed